modules/installation-azure-limits.adoc

```diff
@@ -184,6 +184,14 @@ ifndef::ash[]
 endif::ash[]
 |The internal load balancer, each of the three control plane machines, and each
 of the three worker machines each use a private IP address.
+
```

Comment:
```suggestion
```

I think one of these lines can be deleted safely, but double check.

Reply:
Deleted.